docs/site/RBAC-with-authorization.md

```diff
@@ -47,4 +47,4 @@ The architecture diagram is:
 The example is created in repository
 [access-control-migration](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration),
 and a tutorial of building it from scratch can be found in
-[access control example migration](./migration/auth/example.md).
+[access control example migration](https://loopback.io/doc/en/lb4/migration-auth-access-control-example.html).
```

Comment:
with the convention that the `.md` file name matching the url, I wonder if it's better to rename `example.md` to `migration-auth-access-control-example.md`.